opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/activemq-graphs.properties

```diff
@@ -0,0 +1,109 @@
+reports=activemq.total.consumer.count, \
+activemq.total.connection.count, \
+activemq.message.count, \
+activemq.consumer.producer.count, \
+activemq.storage.count, \
+activemq.temp.storage.count
+
+#
+# Consumer Count
+#
+report.activemq.total.consumer.count.name=ActiveMQ Consumer Count
+report.activemq.total.consumer.count.columns=TtlConsumerCnt
+report.activemq.total.consumer.count.type=interfaceSnmp
+report.activemq.total.consumer.count.command=--title="ActiveMQ Consumer Count" \
+ --units-exponent=0 \
+ --vertical-label="Consumer Count" \
+ DEF:val1={rrd1}:TtlConsumerCnt:AVERAGE \
+ AREA:val1#babdb6 \
+ LINE2:val1#888a85:"Consumer Count " \
+ GPRINT:val1:AVERAGE:"Avg \\: %10.2lf" \
+ GPRINT:val1:MIN:"Min \\: %10.2lf" \
+ GPRINT:val1:MAX:"Max \\: %10.2lf\\n"
+
+#
+# Total Connection Count
+#
+report.activemq.total.connection.count.name=ActiveMQ Total Connection Count
+report.activemq.total.connection.count.columns=TtlConCnt
+report.activemq.total.connection.count.type=interfaceSnmp
+report.activemq.total.connection.count.command=--title="ActiveMQ Connection Count" \
+ --units-exponent=0 \
+ --vertical-label="Connection Count" \
+ DEF:val1={rrd1}:TtlConCnt:AVERAGE \
+ AREA:val1#babdb6 \
+ LINE2:val1#888a85:"Connection Count " \
+ GPRINT:val1:AVERAGE:"Avg \\: %10.2lf" \
+ GPRINT:val1:MIN:"Min \\: %10.2lf" \
+ GPRINT:val1:MAX:"Max \\: %10.2lf\\n"
+
+#
+# Message Count
+#
+report.activemq.message.count.name=ActiveMQ Message Count
```

Comment:
message count vs message size (the same goes for the other labels on this graph)